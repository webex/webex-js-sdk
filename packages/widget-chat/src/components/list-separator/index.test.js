import React from 'react';
import renderer from 'react-test-renderer';

import ListSeparator from '.';

describe(`ListSeparator component`, () => {
  it(`renders properly`, () => {
    const component = renderer.create(
      <ListSeparator
        primaryText={`Testing 123`}
      />
    );
    expect(component).toMatchSnapshot();
  });

  it(`renders properly with classes`, () => {
    const component = renderer.create(
      <ListSeparator
        isInformative
        primaryText={`Testing 123`}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
