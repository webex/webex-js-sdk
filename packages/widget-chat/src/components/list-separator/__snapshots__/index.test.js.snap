exports[`ListSeparator component renders properly 1`] = `
<div
  className="separator">
  <p
    className="separator-text">
    Testing 123
  </p>
</div>
`;

exports[`ListSeparator component renders properly with classes 1`] = `
<div
  className="separator informative">
  <p
    className="separator-text informative">
    Testing 123
  </p>
</div>
`;
